tb_plugin/torch_tb_profiler/profiler/node.py

```diff
@@ -166,6 +171,10 @@ def get_node_argument(event):
         if event.type == EventTypes.KERNEL:
             kwargs["blocks_per_sm"] = event.args.get("blocks per SM", 0)
             kwargs["occupancy"] = event.args.get("est. achieved occupancy %", 0)
+            kwargs["grid"] = str(event.args.get("grid"))
```

Comment:
In case of no these fields, the value will be "None" string. Should we not include these values?

Reply:
Got it. But I prefer still keep the kernel there, and with "None" as this field's value.
I've explicitly set the default value as "None" in latest commit.